packages/core/src/org.ts

```diff
@@ -50,15 +61,15 @@ export class OrgMember extends Serializable {
 
     /** public key of the corresponding [[Account]] */
     @AsBytes()
-    publicKey!: RSAPublicKey;
+    publicKey?: RSAPublicKey;
```

Comment:
These won't be defined until the Member is fully onboarded (i.e. an Account has been created and the invite flow has been completed).